web/css/link.css

```diff
@@ -111,7 +111,7 @@
 
 #wv-link-shorten label {
   position: relative;
-  top: -3px;
+  top: -2px;
```

Comment:
It looks like removing `top` all together aligns things right at least on `chrome` and `firefox`.

Reply:
Weird. I just looked at it in `Browserstack`. Appears to be an `OSX` - `Windows` discrepancy